libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -528,12 +547,24 @@ bool AP_Arming::board_voltage_checks(bool report)
 #if HAL_HAVE_BOARD_VOLTAGE
     // check board voltage
     if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_VOLTAGE)) {
-        if(((hal.analogin->board_voltage() < AP_ARMING_BOARD_VOLTAGE_MIN) || (hal.analogin->board_voltage() > AP_ARMING_BOARD_VOLTAGE_MAX))) {
-            check_failed(ARMING_CHECK_VOLTAGE, report, "Check board voltage");
+        const float bus_voltage =  hal.analogin->board_voltage();
+        if(((bus_voltage < _vbus_min) || (bus_voltage > AP_ARMING_BOARD_VOLTAGE_MAX))) {
+            check_failed(ARMING_CHECK_VOLTAGE, report, "Board voltage (%1.1fv) out of range %1.1f-%1.1fv", bus_voltage, _vbus_min, AP_ARMING_BOARD_VOLTAGE_MAX);
             return false;
         }
     }
 #endif
+
+    if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_VOLTAGE)) {
```

Comment:
We should probably had a new macro for this as the HAL_HAVE_BOARD_VOLTAGE above.